COMPATIBILITY.md

```diff
@@ -0,0 +1,40 @@
+# Instrumented Library Compatibility
+
+Instrumentation is provided for the following Go libraries.
+
+- [`database/sql`](#database/sql)
+- [`github.com/segmentio/kafka-go`](#github.com/segmentio/kafka-go)
+- [`google.golang.org/grpc`](#google.golang.org/grpc)
+- [`net/http`](#net/http)
```

Comment:
Do we need to include the `otelglobal` instrumentation here? or maybe in a separate section about compatibility manual instrumentation.

Reply:
Updated. PTAL.